### ISSUE TITLE
Playlist functionality

### ISSUE THREAD

**Issue by issuemover631** [airsonic/airsonic#459]
<a href="https://github.com/codicusmaximus"><img src="https://avatars1.githubusercontent.com/u/11328458?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [codicusmaximus](https://github.com/codicusmaximus)**
_Thursday Jun 08, 2017 at 17:59 GMT_
_Originally opened as https://github.com/Libresonic/libresonic/issues/459_

----

Does anyone use the playlist functionality?  If so, in what way?  I don't know if my needs are just different from the rest of the world or what but I find using playlists at all to be way too clunky to get any real use out of them.  I would ideally want to right click a song or click a context button to the right of the song in the list to get a small popup with a search box at the top and 10 most recently added-to playlists listed to click to add to one of them or search for the one to add to.  Also maybe even just a single button to insta-add to the current/most-recently-edited playlist.

Unless there's an easier way I'm not aware of, to add a song I find while browsing around to a "Song with epic bass drops" playlist, i have to add it to the play queue, go into the play queue, check-box the song, click into the operations dropdown to select 'add to playlist', select the playlist.

**Comment by Guruleenyc** [comment]
Agreed! Also so annoying that you cannot just simply add current song playing to a an existing, or create a new playlist if one doesn't exist-------------NOT all queue :-/
